kong/runloop/handler.lua

```diff
@@ -1450,16 +1446,6 @@ return {
       if var.http_proxy_connection then
         clear_header("Proxy-Connection")
       end
-
-      -- X-Kong-Request-Id upstream header
```

Comment:
I think that we should add some comments to tell the reviewer that the set_header logic is moved to nginx-kong template.

Reply:
OK!

Reply:
hi @chronolaw Even though I think I can add a comment here, but it is like strange, because

1. I think this tunning patch is part of request_id feature. If I assume that I'm adding this feature from scratch, we probably shouldn't add a comment in an extra code location (like this one) to say that we will add a comment in nginx-kong.conf explaining the addition of headers.
2. There are so many upstream headers appended in nginx-kong.conf, if we add the comment for x_kong_request_id here, I think we need to add comments for all other headers.

So I think its unnecessary to add comment here. Other people can get this info from this commit log